Cargo.toml

```diff
@@ -34,7 +34,12 @@ byteorder = "1.4.3"
 thiserror = "1.0"
 halo2curves = { version = "0.4.0", features = ["derive_serde"] }
 group = "0.13.0"
+pairing = "0.23.0"
```

Comment:
Why is this necessary? It does not seem to be used. Isn't it?

Reply:
Indeed, my bad 🤦